bundles/org.openhab.binding.boschindego/src/main/java/org/openhab/binding/boschindego/internal/handler/BoschIndegoHandler.java

```diff
@@ -55,16 +59,22 @@ public class BoschIndegoHandler extends BaseThingHandler {
     private final Logger logger = LoggerFactory.getLogger(BoschIndegoHandler.class);
     private final HttpClient httpClient;
     private final BoschIndegoTranslationProvider translationProvider;
+    private final TimeZoneProvider timeZoneProvider;
 
     private @NonNullByDefault({}) IndegoController controller;
-    private @Nullable ScheduledFuture<?> pollFuture;
-    private long refreshRate;
+    private @Nullable ScheduledFuture<?> statePollFuture;
+    private @Nullable ScheduledFuture<?> cuttingTimePollFuture;
+    private long stateRefreshRate;
+    private long cuttingTimeRefreshRate;
```

Comment:
Apparently, you don't really class members for `stateRefreshRate` and `cuttingTimeRefreshRate`. Simple local variables would be enough.

Reply:
You are completely right - fixed.